package/upgrade/upgrade_manifests.sh

```diff
@@ -94,9 +245,16 @@ EOF
   upgrade_managed_chart_from_version $UPGRADE_PREVIOUS_VERSION harvester harvester.yaml
   NEW_VERSION=$REPO_HARVESTER_CHART_VERSION yq e '.spec.version = strenv(NEW_VERSION)' harvester.yaml -i
   kubectl apply -f ./harvester.yaml
+
+  pause_managed_chart harvester "false"
+  pause_managed_chart harvester-crd "false"
+
```

Comment:
Would be nice to wait managed chart to ready here: please see https://github.com/bk201/harvester/commit/314ab334d80a7cb9a7417e52bd4c1a9339c403fb#diff-8c5fdd3c29c4031d6332031bafd4f1c97820a492be51384b2b22689c6322d31fR267-R268